docs/src/pages/components/material-icons/SearchIcons.js

```diff
@@ -0,0 +1,480 @@
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions, jsx-a11y/click-events-have-key-events */
+
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Paper from '@material-ui/core/Paper';
+import clsx from 'clsx';
+import InputBase from '@material-ui/core/InputBase';
+import Typography from '@material-ui/core/Typography';
+import PropTypes from 'prop-types';
+import { debounce } from 'lodash';
+import Grid from '@material-ui/core/Grid';
+import FormGroup from '@material-ui/core/FormGroup';
+import Dialog from '@material-ui/core/Dialog';
+import MarkdownElement from 'docs/src/modules/components/MarkdownElement';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import IconButton from '@material-ui/core/IconButton';
+import Tooltip from '@material-ui/core/Tooltip';
+import Button from '@material-ui/core/Button';
+import FlexSearch from 'flexsearch';
+import SearchIcon from '@material-ui/icons/Search';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Checkbox from '@material-ui/core/Checkbox';
+import Link from 'docs/src/modules/components/Link';
+import * as mui from '@material-ui/icons';
+// import Menu from '@material-ui/icons/Menu';
```

Comment:
Are these needed?

Reply:
I have used them to work on the logic. Rendering 1k SVG icons and importing 5k can be slow in dev mode. It's mostly here so we can quickly iterate in the future. I don't have a strong preference. I know that I can get them back from another demo. What do think?

Reply:
I don't have a strong preference either. It might be worth adding a comment to just mention they are there to allow for quick iteration though.